python/cudf/cudf/io/json.py

```diff
@@ -54,6 +54,22 @@ def _get_cudf_schema_element_from_dtype(
     return lib_type, child_types
 
 
+def _to_plc_compression(
```

Comment:
Do we use something like this in other places? If so, we could reuse it.

Reply:
It looks like we have something similar we use in `parquet`, but given that each format supports different compression (and maps slightly differently from Python), I can take a look at this in a follow up